apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerParameterCollection.cs

```diff
@@ -29,6 +29,23 @@ namespace Google.Cloud.Spanner.Data
     /// </summary>
     public sealed class SpannerParameterCollection : DbParameterCollection, IEnumerable<SpannerParameter>
     {
+        /// <summary>
+        /// Initializes a new SpannerParameterCollection
+        /// </summary>
+        public SpannerParameterCollection()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new SpannerParameterCollection with the given set of initial <see cref="SpannerParameter"/>
+        /// objects.
+        /// </summary>
+        /// <param name="initialParameters">The initial parameters to include in the collection.</param>
+        public SpannerParameterCollection(IEnumerable<SpannerParameter> initialParameters)
```

Comment:
Make this default to null perhaps and then just add a null check below? Then only one constructor is needed.